src/visualization/interaction/OrbitControls.js

```diff
@@ -14,7 +20,9 @@
  * @param userZoomSpeed (optional) - the speed for zooming
  * @param userRotateSpeed (optional) - the speed for rotating
  * @param autoRotate (optional) - if the orbit should auto rotate
- * @param autoRotate (optional) - the speed for auto rotating
+ * @param buttonRotate (optional) - the mouse button for rotating the view (defaults to LEFT)
+ * @param buttonMove (optional) - the mouse bitton for moving the view (defaults to MIDDLE)
+ * @param buttonZoom (optional) - the mouse button for zooming the view (defaults to RIGHT)
```

Comment:
I think it is more make sense to configure all three all at once instead of configuring them individually. How about making it as `buttonEvent` param and receive a object to configure all three. e.g) `buttonEvent : {'buttonRotate': ROS3D.MOUSE_BUTTON.LEFT, 'buttonMove':ROS3D.MOUSE_BUTTON.MIDDLE ....}` Or do you prefer to have them as separate parameter?

Also, It would be nice to check mutual exclusivity to avoid accidental mapping two events into one key binding.

Reply:
Sure, I'll try to finish this in the next couple of days... I'll also add some tests if time allows. Sorry it took me so long.. :)